docs/portal_config.md

```diff
@@ -388,7 +388,17 @@ Below is an example, with inline comments describing what each JSON block config
     "dropdowns": {} // optional; dropdown groupings for buttons
   },
   "discoveryConfig": { // config for Discovery page. Required if 'featureFlags.discovery' is true. See src/Discovery/DiscoveryConfig.d.ts for Typescript schema.
+    "requireLogin": false, // optional, defaults to false. If true, requires user to sign in before seeing the Discovery page
```

Comment:
the `Resource Browser` component is calling this field `public`, we should do that, we already have `indexPublic`, `explorerPublic` and `resourceBrowserPublic` done in 3 different ways, we should avoid adding 4th way if possible

Reply:
Haha great point, I'll update this to be `public: boolean` instead of `requireLogin: boolean`